ipfs-cluster-service/main.go

```diff
@@ -356,15 +371,14 @@ func initConfig(force bool, generateSecret bool, envSecret string) {
 	cfg, err := ipfscluster.NewDefaultConfig()
 	checkErr("creating default configuration", err)
 
-	if !generateSecret {
-		if len(envSecret) != 0 {
-			// read cluster secret from env variable
-			fmt.Println("Reading cluster secret from CLUSTER_SECRET environment variable.")
-			cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(envSecret)
-		} else {
-			// get cluster secret from user
-			cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(promptUser("Enter cluster secret (to automatically generate, rerun with --gen-secret): "))
-		}
+	if len(envSecret) != 0 {
+		// read cluster secret from env variable
+		fmt.Println("Reading cluster secret from CLUSTER_SECRET environment variable.")
+		cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(envSecret)
+	} else if !generateSecret {
+		// get cluster secret from user
+		cfg.ClusterSecret, err = ipfscluster.DecodeClusterSecret(promptUser("Enter cluster secret (to automatically generate, rerun with --gen-secret): "))
+
 		checkErr("parsing cluster secret", err)
```

Comment:
Move this below bottom brace, currently doesn't check the `err` from the first `if` case.